src/test/scanning/scan_test.ts

```diff
@@ -23,7 +23,7 @@ import {invertPromise} from '../test-utils';
 suite('scan()', () => {
 
   test('calls Scanner.scan', async() => {
-    const feature = Symbol('feature');
+    const feature = Symbol('feature') as any;
```

Comment:
Yep. Same with the dependency graph tests, where we treat strings as ParsedDocuments for easy comparison. I'm fine with doing a more comprehensive fix later but this is just to unblock